i18n/dnd5e/gl/dnd5e.json

```diff
@@ -0,0 +1,758 @@
+  "DND5E.NoSpellLevels": "Este personaxe non posúe níveis como conxurador, pero podes engadir maxias manualmente.",
+  "DND5E.FlagsInstructions": "Configure características e trazos de personaxes que axustam comportamentos do sistema DnD5e.",
+  "DND5E.FlagsSave": "Actualice Trazos Especiais",
+  "DND5E.FlagsTitle": "Configure Trazos Especiais",
+  "DND5E.FlagsDiamondSoul": "Alma de Diamante",
+  "DND5E.FlagsDiamondSoulHint": "Gaña proficiencia en todas as Salvagardas.",
+  "DND5E.FlagsPowerfulBuild": "Forma Poderosa",
+  "DND5E.FlagsPowerfulBuildHint": "Prové capacidade de carga superior.",
+  "DND5E.FlagsElvenAccuracy": "Precisión Élfica",
+  "DND5E.FlagsElvenAccuracyHint": "Role un d20 extra cando posúe vantaxe en proba de Des, Int, Sab ou Car.",
+  "DND5E.FlagsHalflingLucky": "Mediano: Sortudo",
+  "DND5E.FlagsHalflingLuckyHint": "Rola novamente os 1s cando faga roladas con d20.",
+  "DND5E.FlagsInitiativeAdv": "Vantaxe en Iniciativa",
+  "DND5E.FlagsInitiativeAdvHint": "De características do personaxe ou items máxicos.",
+  "DND5E.FlagsAlert": "Talento: Alerta",
+  "DND5E.FlagsAlertHint": "Prové +5 en Iniciativa.",
+  "DND5E.FlagsJOAT": "Pau para Toda Obra",
+  "DND5E.FlagsJOATHint": "Media proficiencia para Probas de Atributo en que non sexas proficiente.",
+  "DND5E.FlagsObservant": "Talento: Analítico",
+  "DND5E.FlagsObservantHint": "Prové un bonus de +5 para a Percepción pasiva e Investigación pasiva.",
+  "DND5E.FlagsReliableTalent": "Talento Confiábel",
+  "DND5E.FlagsReliableTalentHint": "Característica Talento Confiábel de Ladinos.",
+  "DND5E.FlagsRemarkableAthlete": "Atleta Extraordinario.",
+  "DND5E.FlagsRemarkableAthleteHint": "Adiciona media Proficiencia (arredondada cara arriba) en Iniciativa e Probas de Atributos físicos.",
+  "DND5E.FlagsWeaponCritThreshold": "Intervalo de Acerto Crítico con Armas",
+  "DND5E.FlagsWeaponCritThresholdHint": "Un intervalo de acerto crítico expandido para ataques con armas.",
+  "DND5E.FlagsSpellCritThreshold": "Intervalo de Acerto Crítico con Maxias",
+  "DND5E.FlagsSpellCritThresholdHint": "Un intervalo de acerto crítico expandido para ataques máxicos.",
+  "DND5E.FlagsMeleeCriticalDice": "Dado de dano crítico corpo a corpo",
+  "DND5E.FlagsMeleeCriticalDiceHint": "O número de dados de dano extra adicionados ao acerto crítico de armas corpo a corpo.",
+  "DND5E.Flat": "Fixo",
+  "DND5E.Formula": "Fórmula",
+  "DND5E.GrantedAbilities": "Habilidades Adquiridas",
+  "DND5E.HalfProficient": "Medio Proficiente",
+  "DND5E.Healing": "Cura",
+  "DND5E.HealingTemp": "Cura (Temporal)",
+  "DND5E.HitPoints": "Puntos de Golpe",
+  "DND5E.HP": "PV",
+  "DND5E.HealthConditions": "Condicións de Puntos de Golpe",
+  "DND5E.HPFormula": "Fórmula de Puntos de Golpe",
+  "DND5E.HitDice": "Dados de Golpe",
+  "DND5E.HitDiceRoll": "Rolar Dados de Golpe",
+  "DND5E.HitDiceUsed": "Dados de Golpe utilizados",
+  "DND5E.HitDiceWarn": "{name} non posúe dados de golpe {formula} remanescentes!",
```

Comment:
Eu o de "remanescente" nunca o escoitei, non o atopo tampouco no dicionario da RAG. Talvez mellor algo tipo "A {name} non lle quedan dados de golpe {formula}"?

Reply:
Si, ou "restantes" ou algo máis simple como iso que propós. Ficou aí do português.